src/startup.rs

```diff
@@ -61,16 +68,17 @@ pub fn setup_panic_hook(reporting_enabled: bool, reporting_id: String) {
 /// Creates a file that indicates that the server has been started.
 /// This file is used to check if the server has been been successfully started before potential kill.
 pub fn touch_started_file_indicator() {
-    if let Err(err) = std::fs::write(INITIALIZED_FILE, "") {
+    if let Err(err) = std::fs::write(get_init_file_path(), "") {
         log::warn!("Failed to create init file indicator: {}", err);
     }
 }
 
 /// Removes a file that indicates that the server has been started.
 /// Use before server initialization to avoid false positives.
 pub fn remove_started_file_indicator() {
-    if std::path::Path::new(INITIALIZED_FILE).exists() {
-        if let Err(err) = std::fs::remove_file(INITIALIZED_FILE) {
+    let path = get_init_file_path();
```

Comment:
Do we have a "migration"-issue here? In case somebody changes this path via an ENV var between two runs, the old file would not be cleaned up. The old file would not be considered for any logic internally, since the path changed but it still remains on disk. So this is the obligation of the caller to delete irritating old files then?

Reply:
Correct. But I think that is impossible to fix without using a common prefix.

Reply:
is it worth to have a one sentence documentation addition somewhere?

Reply:
Add docs for the user to be careful, or to describe when this file is created and cleaned up?

In both cases: sure!